app/Http/Controllers/UserController.php

```diff
@@ -12,11 +12,30 @@ class UserController extends ApiController
      *
      * @return \Illuminate\Http\Response
      */
-    public function index(UserStore $userStore)
+    public function index(UserStore $userStore, Request $request)
     {
+        $this->setFiltersParameters($request);
         $users = $userStore->all();
+
+        if ($this->status) {
+            $users = $this->getUsersByStatus($users, $this->status);
+        }
+
         return $this->setStatusCode(200)->respond($users);
     }
+
+    public function getUsersByStatus($users, $status) {
+        $filteredUsers = [];
+
+        foreach ($users as $user) {
+            if ($user['status'] == $status) {
+                array_push($filteredUsers, $user);
+            }
+        }
```

Comment:
FYI: PHP has awesome function for this action: array_filter(); So you can use it next time.